Rakefile

```diff
@@ -1,27 +1,5 @@
-# -*- ruby -*-
+require 'bundler/setup'
+require 'bundler/gem_tasks'
+require 'rake/testtask'
 
-require 'rubygems'
-require 'hoe'
-
-Hoe.plugin :git
-Hoe.plugin :minitest
-Hoe.plugin :travis
-
-Hoe.spec 'net-http-persistent' do
-  developer 'Eric Hodel', 'drbrain@segment7.net'
-
-  self.readme_file      = 'README.rdoc'
-  self.extra_rdoc_files += Dir['*.rdoc']
-
-  self.require_ruby_version '~> 2.1'
-
-  license 'MIT'
-
-  rdoc_locations <<
-    'docs.seattlerb.org:/data/www/docs.seattlerb.org/net-http-persistent/'
```

Comment:
The docs are no longer published and I don't think bundler's publishing tasks have this capability for a `rake release`.

Reply:
can we use a more standard docs host ... or just remove hosting docs in favor of directing people to just read the docs/readme on github ? (removes lots of complications ... and this is a tiny gem anyway, so no need for fancy docs)

Reply:
... or leave that for "whenever someone care enough to fix it" ... because I'd love to actually fix the code and not be blocked by this docs issue :(

Reply:
> so no need for fancy docs

I spent a lot of time writing comprehensive documentation that's accessible from standard tools.  Pretending like they don't exist and that they'll maintain themselves means they'll decay.  

Reading documentation out of source files isn't discoverable or easy for people who aren't experienced rubyists or aren't familiar with the library.  Having a first-class experience as an introduction to a library is essential.

I've fixed the docs issue by fixing hoe-travis (1.4.1 release) and creating seattlerb/hoe#89.  The tests now run and pass on master on travis.